snapcraft_legacy/plugins/v2/_kernel_build.py

```diff
@@ -278,17 +278,20 @@ def _do_base_config_cmd(
         logger.info("Using ubuntu config flavour %s", config_flavour)
         conf_cmd = textwrap.dedent(
             """	echo "Assembling Ubuntu config..."
-	branch=$(cut -d'.' -f 2- < ${{KERNEL_SRC}}/debian/debian.env)
-	baseconfigdir=${{KERNEL_SRC}}/debian.${{branch}}/config
-	archconfigdir=${{KERNEL_SRC}}/debian.${{branch}}/config/${{DEB_ARCH}}
-	ubuntuconfig=${{baseconfigdir}}/config.common.ubuntu
-	archconfig=${{archconfigdir}}/config.common.${{DEB_ARCH}}
-	flavourconfig=${{archconfigdir}}/config.flavour.{config_flavour}
-	cat ${{ubuntuconfig}} ${{archconfig}} ${{flavourconfig}} \
-> {dest_dir}/.config 2>/dev/null || true
 	if [ -f ${{KERNEL_SRC}}/debian/rules ] && [ -x ${{KERNEL_SRC}}/debian/rules ]; then
+		# Generate Ubuntu kernel configs
+		pushd ${{KERNEL_SRC}}
+		fakeroot debian/rules clean genconfigs || true
```

Comment:
we should improve upstream kernel packaging to not require fakeroot for all of the builds.

Is fakeroot installed/available at this point of the build? I.e. is there somewhere build-depends that installs fakeroot and things for genconfigs to run?

Reply:
Yes, snapcraft recipes using this plugin have a `build-depends` specifying `fakeroot`.

Reply:
@kubiko's comment is valid, until fakeroot is no longer needed

Reply:
this is resolved then.

Reply:
@sergiusens please dismiss this review comment, which I raised, but cannot mark as resolved.